docs/user-guide/query/gaffer-syntax/filtering.md

```diff
@@ -654,6 +654,347 @@ total for all the `added` and `removed` properties.
+
+        ```json
+        {
+            "class": "GetElements",
+            "input": [
+                {
+                    "class": "EntitySeed",
+                    "vertex": "John"
+                }
+            ],
+            "view": {
+                "edges": {
+                    "Commit": {
+                        "groupBy" : [ ],
+                        "preAggregationFilterFunctions": [{
+                            "selection" : ["date"],
+                            "predicate" : {
+                                "class" : "InDateRange",
+                                "start" : "2024/05/01",
+                                "end" : "2024/05/03",
+                                "startInclusive": true,
+                                "endInclusive": false
+                            }
+                        }]
+                    }
+
+                }
+            }
+        }
+        ```
+
+    === "Python"
+
+        ```python
+        elements = g_connector.execute_operation(
+            operation = g.GetElements(
+                input = [g.EntitySeed(vertex = "John")]
+                view = g.View(
+                    edges = [
+                        g.ElementDefinition(
+                            group = 'Commit',
+                            group_by = [],
+                            pre_aggregation_filter_functions=[
+                                g.PredicateContext(
+                                    predicate=g.Or(
+                                        predicates=[
+                                            g.InDateRange(
+                                                start="2024/05/01",
+                                                end="2024/05/03",
+                                                start_inclusive=True,
+                                                end_inclusive=False
+                                            )
+                                        ]
+                                    )
+                                )
+                            ]
+                        )
+                    ]
+                )
+            )
+        )
+        ```
+
+    Result:
+
+    === "Java"
+        ```java
+        Edge[source="John",destination=1,directed=true,matchedVertex=SOURCE,group=Commit,properties=Properties[date=<java.util.Date>Wed May 01 00:00:00 UTC 2024, count=<java.lang.Long>2]]
+        ```
+
+There is also a more advanced feature of query-time aggregation which allows you to override
+the logic of how Gaffer aggregates properties. For example, by default the count property is aggregated
+with Sum, as above, but at query time you can force the count to be aggregated using another operator,
+e.g. the Min operator. This would allow you to find things such as the minimum daily count.
+This feature does not affect any of the persisted values and any ingest aggregation that has already
+occurred will not be modified.
+
+??? example
```

Comment:
All the other examples are expanded might want to change this to `!!! example ""`